superset-frontend/plugins/legacy-preset-chart-deckgl/src/DeckGLContainer.jsx

```diff
@@ -64,6 +64,12 @@ export class DeckGLContainer extends React.Component {
     };
   }
 
+  UNSAFE_componentWillReceiveProps(nextProps) {
+    if (nextProps.viewport !== this.props.viewport) {
```

Comment:
I believe this will cause unnecessary rerenders as the object reference may change despite its properties being unchanged. Let's use `lodash` here, something like
```javascript
import { isEqual } from 'lodash';

if (!isEqual(nextProps.viewport, this.props.viewport)) {
  ...
},
```

Reply:
Thanks for the hint. Is adressed by last commit (though checks failed again)...